modules/p2/ding_list/theme/theme.inc

```diff
@@ -164,24 +165,33 @@ function theme_ding_list_lists($variables) {
       }
     }
 
-    $out .= theme('item_list', array(
-      'items' => $rows,
-      'title' => $group['title'],
-      'type' => 'ul',
-      'attributes' => array(),
-    ));
+    if (!empty($rows)) {
+      $render = array(
+        '#theme' => 'item_list',
+        '#title' => $group['title'],
+        '#items' => $rows,
+        '#title' => $group['title'],
+        '#type' => 'ul',
+        '#attributes' => array(),
+      );
+      $out .= drupal_render($render);
```

Comment:
It's called from inside a theme_HOOK() function, we could move the creation of the item_list to a preprocessor, but we'd still need to call drupal_render inside the theme_HOOK().